taskcluster/scripts/build/wasm.sh

```diff
@@ -12,18 +12,28 @@ qmake --version
 git submodule init
 git submodule update
 
+wget https://github.com/Kitware/CMake/releases/download/v3.24.0-rc4/cmake-3.24.0-rc4-linux-x86_64.sh
+chmod +x cmake-3.24.0-rc4-linux-x86_64.sh
+mkdir /tmp/cmake
+./cmake-3.24.0-rc4-linux-x86_64.sh --skip-license --prefix=/tmp/cmake
```

Comment:
NIT: I think this should go here :)
 https://github.com/mozilla-mobile/mozilla-vpn-client/blob/main/taskcluster/docker/wasm/Dockerfile